### DISCUSSION TITLE
Return code from cdk deploy and bootstrap commands

### DISCUSSION THREAD

**Discussion by ganeshnj** [aws/aws-cdk#20268] [Q&A]
We are trying to understand the errors that can happen while running `cdk deploy`, `cdk bootstrap` or any other cdk command, what is the expected exit code customer can expect other than 0 (success).

Holistically, I would like to know list of exit codes returned by CDK CLI. Is there a glossary that we can refer to?

**Comment by peterwoodworth** [comment]
There are deployment errors, which are errors directly returned from CloudFormation API calls. And there are synth-time errors, which should come with a clear error message. We don't use error codes as far as I'm aware
